### ISSUE TITLE
Issues with creating table in production

### ISSUE THREAD

**Issue by Globegitter** [balderdashy/sails-mysql#129]
Just trying to run my sails app in production and when I am accessing one of my models `/api/v1/forms/` I get the following error:

![capture decran 2014-07-25 a 17 08 18](https://cloud.githubusercontent.com/assets/2580254/3704610/0a36454c-1416-11e4-869f-c981fdf2c884.png)

It seems to be having issues with creating the table when running sails lift. No idea what the issue is, since it has worked before in my development environemnt. Any ideas what could be wrong? Trying to run with mongodb for now since I need this in production now.

Edit: Just run with the `--silly` flag and seeing:

![capture decran 2014-07-25 a 17 22 26](https://cloud.githubusercontent.com/assets/2580254/3704792/f002a614-1417-11e4-92eb-a245da7efc85.png)

Aahhh, so according to http://stackoverflow.com/questions/20490853/stop-the-auto-migration-of-the-schema-in-the-sailsjs `migrate: 'safe'` is the issue. So how do I run migrations then? http://beta.sailsjs.org/#/documentation/concepts/Deployment doesn't mention anything about that.

**Comment by marcusnielsen** [comment]
This bothered me a lot.

I didn't even believe the documentation after reading this advice about creating a local database and then dumping to remote prod. It feels hacky.

Suggestions:

1. There is a config where I can set `migrate: 'alter'`. That should not be overridden, just be a default. You can print a warning instead of forcing behavior as with the session store.

2. Give an option to have safeStrict or safeToAdd for prodution setup of empty DBs.

I'm a bit stressed right now, so sorry for the short tone :-)

**Comment by sailsbot** [comment]
Thanks for posting, @Globegitter.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
